articles/media-services/latest/configure-connect-java-howto.md

```diff
@@ -70,28 +70,26 @@ When you run the command, the `pom.xml`, `App.java`, and other files are created
 1. Under the package statement, add these import statements:
    
    ```java
-   import com.microsoft.azure.AzureEnvironment;
-   import com.microsoft.azure.credentials.ApplicationTokenCredentials;
-   import com.microsoft.azure.management.mediaservices.v2018_07_01.implementation.MediaManager;
-   import com.microsoft.rest.LogLevel;
+   import com.azure.core.management.AzureEnvironment;
+   import com.azure.core.management.profile.AzureProfile;
+   import com.azure.identity.ClientSecretCredential;
+   import com.azure.identity.ClientSecretCredentialBuilder;
+   import com.azure.resourcemanager.mediaservices.MediaServicesManager;
    ```
 1. To create the Active Directory credentials that you need to make requests, add following code to the main method of the App class and set the values that you got from [Access APIs](./access-api-howto.md):
    
    ```java
-   final String clientId = "00000000-0000-0000-0000-000000000000";
-   final String tenantId = "00000000-0000-0000-0000-000000000000";
-   final String clientSecret = "00000000-0000-0000-0000-000000000000";
-   final String subscriptionId = "00000000-0000-0000-0000-000000000000";
-
    try {
-      ApplicationTokenCredentials credentials = new ApplicationTokenCredentials(clientId, tenantId, clientSecret, AzureEnvironment.AZURE);
-      credentials.withDefaultSubscriptionId(subscriptionId);
-
-      MediaManager manager = MediaManager
-              .configure()
-              .withLogLevel(LogLevel.BODY_AND_HEADERS)
-              .authenticate(credentials, credentials.defaultSubscriptionId());
-      System.out.println("signed in");
+        AzureProfile azureProfile = new AzureProfile(AzureEnvironment.AZURE);
```

Comment:
If given a coded input (instead of env variable), this line better to be:
```
        AzureProfile azureProfile = new AzureProfile("<YOUR_TENANT_ID>", "<YOUR_SUBSCRIPTION_ID>", AzureEnvironment.AZURE);
```

Reply:
if we use this approach, will the ClientSecretBuilder change?

Reply:
No, it will not change.

Basically, auth requires `tenantId` (in addition to `clientId` and `clientSecret`).

Mgmt SDK itself requires `subscriptionId` (and some feature also requires `tenantId`).

Reply:
got it, thanks, i will make the change